osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -177,22 +189,36 @@ protected override void Update()
         {
             base.Update();
 
-            Vector2? lowestTopScreenSpace = null;
+            float? lowestTopScreenSpaceLeft = null;
+            float? lowestTopScreenSpaceRight = null;
+
             Vector2? highestBottomScreenSpace = null;
 
             // LINQ cast can be removed when IDrawable interface includes Anchor / RelativeSizeAxes.
             foreach (var element in mainComponents.Components.Cast<Drawable>())
             {
                 // for now align top-right components with the bottom-edge of the lowest top-anchored hud element.
```

Comment:
I guess this should be reworded to:
```suggestion
                // for now align top components with the bottom-edge of the lowest top-anchored hud element.
```
Assuming the condition was updated to have the leaderboard position itself below top-left components as well?